setup.py

```diff
@@ -69,6 +69,7 @@ def package_tree(pkgroot):
                        'Programming Language :: Python :: 3',
                        ],
           platforms='any',
+          install_requires=['numpy>=1.11.3', 'scipy>=0.17.1'],
```

Comment:
Someday we could probably parse this from `README.rst` to make it DRY but don't have to do it now

Reply:
Agreed, we should have only one place where we specify the exact versions.